spread.yaml

```diff
@@ -199,6 +202,9 @@ prepare: |
         if [ -d /etc/apt/apt.conf.d ] && [ -n "${APT_PROXY:-}" ]; then
             printf 'Acquire::http::Proxy "%s";\n' "$APT_PROXY" > /etc/apt/apt.conf.d/99proxy
         fi
+        # XXX: This seems to be required on 14.04. Otherwise package build
+        # fails with unmet dependency on "build-essential:native"
+        apt-get install -y build-essential
```

Comment:
I suspect this is because gdebi (that is used to resolve the build-deps in 14.04) is not quite as good as `apt build-dep  .`. If it is only needed on 14.04, I wonder if we can move it into the 14.04 specific `if` so that we have all the "hacks" in a single place?